geetools/Image.py

```diff
@@ -1329,23 +1347,87 @@ def maskCover(self) -> ee.Image:
+        values = ee.Dictionary(cover.values().get(0))
+        zeros, ones = ee.Number(values.get("0", 0)), ee.Number(values.get("1", 0))
+        ratio = ones.divide(zeros.add(ones)).multiply(100)
+
+        # we want to display this result as a 1 digit float
+        return ratio
+
+    def maskCoverRegions(
+        self,
+        collection: ee.FeatureCollection,
+        scale: Optional[int] = None,
+        band: Optional[str] = None,
+        proxy_value: int = -999,
+        column_name: str = "mask_cover",
+        tileScale: int = 1,
+    ) -> ee.FeatureCollection:
+        """Compute the coverage of masked pixels inside a Geometry.
+
+        Parameters:
+            collection: The collection to compute the mask coverage (in each Feature).
+            scale: The scale of the computation. In case you need a rough estimation use a higher scale than the original from the image.
+            band: The band to use. Defaults to the first band.
+            proxy_value: the value to use for counting the mask and avoid confusing 0s to masked values. Choose a value that is out of the range of the image values.
+            column_name: name of the column that will hold the value.
+            tileScale: A scaling factor between 0.1 and 16 used to adjust aggregation tile size; setting a larger tileScale (e.g., 2 or 4) uses smaller tiles and may enable computations that run out of memory with the default.
+
+        Returns:
+            The passed table with the new column containing the percentage of masked pixels within the region
+
+        Examples:
+            .. code-block:: python
+
+                    import ee, geetools
+
+                    ee.Initialize()
+
+                    image = ee.Image('COPERNICUS/S2_SR/20190828T151811_20190828T151809_T18GYT')
+                    reg = ee.Geometry.Point([11.880190936531116, 42.0159494554553]).buffer(2000)
+                    aoi = ee.FeatureCollection([ee.Feature(reg)])
+                    image = image.maskCoverRegions(aoi)
+        """
+        # compute the mask cover
+        properties = collection.propertyNames()  # original properties
+        image = self._obj.select(band or 0)
+        scale = scale or image.projection().nominalScale()
+        unmasked = image.unmask(proxy_value)
+        mask = unmasked.eq(proxy_value)
+        column = "_geetools_histo_"
+        cover = mask.reduceRegions(
+            collection=collection,
+            reducer=ee.Reducer.frequencyHistogram().setOutputs([column]),
+            scale=scale,
+            tileScale=tileScale,
+        )
 
-        # we want to display this resutl as a 1 digit float
-        ratio = ratio.multiply(1000).toInt().divide(10)
+        def compute_percentage(feat: ee.Feature) -> ee.Feature:
+            histo = ee.Dictionary(feat.get(column))
+            zeros, ones = ee.Number(histo.get("0", 0)), ee.Number(histo.get("1", 0))
+            ratio = ones.divide(zeros.add(ones)).multiply(100)
```

Comment:
same here can we keep the presision of the ratio to 2 decimals ? 
```suggestion
            ratio = ones.divide(zeros.add(ones)).multiply(10000).int().divide(100)
```

Reply:
same as above